### PR TITLE
Get Active Tasks

### PR DESCRIPTION
**Pull request by joratrifork**

### CODE REVIEW

File: openapi/schemas/GetTimeRegistrationsByTaskRequest.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+type: array
```
Comment (maratrifork) [inline openapi/schemas/GetTimeRegistrationsByTaskRequest.yaml:1]: Shouldn't this be a _TimeRegistrationsByTask**Response**._ ?

File: openapi/active-task-registrations.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+get:
+  tags:
+    - "Time registration"
+  operationId: "getTimeRegistrationsByActiveTaskForUser"
```
Comment (maratrifork) [inline openapi/active-task-registrations.yaml:4]: Don't know if i love the name, should we ditch "ForUser" in general for our endpoints? If it's implicitly for the current user, then it's kind of superfluous.

This operation is something about an overview by task, so maybe `getTaskTimeRegistrationsOverview`?

File: openapi/schemas/GetTimeRegistrationsByTaskRequest.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      type: string
+    taskName:
+      type: string
+    taskDescription:
+      type: string
+    timeRegistrations:
+      type: array
+      items:
+        type: "object"
+        required: [ "timeRegistrationId", "registered", "date", "duration", "status" ]
```
Comment (maratrifork) [inline openapi/schemas/GetTimeRegistrationsByTaskRequest.yaml:15]: I know you moved the `taskId` to the top level, since it's the dame for all. Will it make it harder to create clients that this type is unrelated to the existing `TimeRegistrationResponse`?

I'm not saying we should always use the same types across all endpoints, and it might be better to keep them separate for now.

Comment (joratrifork) [reply openapi/schemas/GetTimeRegistrationsByTaskRequest.yaml:15]: > I know you moved the taskId to the top level, since it's the dame for all. Will it make it harder to create clients that this type is unrelated to the existing `TimeRegistrationResponse`?

I think that will depend on how the client generates the types (if they do).

Generally for API stuff, I prefer keeping request/response types separate (even though they might be identical to another request/response type) as you never know if a future version will add or remove a property.
